### DISCUSSION TITLE
"Reference in new discussion" feature not working when categories are confgured

### DISCUSSION THREAD

**Discussion by Alwaysin** [community/community#122446] [Discussions]
### Select Topic Area

Bug

### Body

When using said feature, if the repository is configured with categories you have to choose from, the URL loses the permalink, from `https://github.com/<org>/<repo>/discussions/new?permalink=xxx` to a generic `https://github.com/<org>/<repo>/discussions/new/choose`, and after making the choice, nothing is referenced.
An error appears on the selection page:
```
Sorry, we didn't recognize that category! Please choose one of the following valid categories.
```

**Comment by Rickyc81** [comment]
and so? any solutions?
